homeassistant/components/tradfri/light.py

```diff
@@ -143,99 +144,55 @@ def _observe_update(self, tradfri_device):
         await self._api(self._group.update())
 
 
-class TradfriLight(Light):
+class TradfriLight(Light, TradfriBaseDevice):
```

Comment:
Should we change the inheritance order and put the `Light` class last? It's usually better to put the custom class closer to the child class, eg since we implement an init method in the custom class.

Reply:
I've always wondered if the order of inheritance matters but I'll remember that as a rule of thumb. Just pushed an update with the proposed change.

Reply:
It doesn't matter here yet, but it's more future proof this way.